data/hai/hai reveal 5 philosophers.txt

```diff
@@ -1011,84 +1011,129 @@ mission "Hai Reveal [B13] Freedom"
+			`	"Your question was wise. There is one unaccounted for. Before I tell you, though, I think perhaps you should know more than you do." They reach up and push back their hood, revealing a bald head, as they gesture to the city in general. To your surprise you realize for the first time that they are a woman, though that seems like it could be the least important part of their identity.`
+			choice
+				`	"What do you wish to tell me?"`
+				`	"I've had enough history lessons for one day. Can you tell me the leader's name?"`
+					goto impatient
+			`	"Many different overlords we have had, and always they have relied on recruits from the city for some capacities, and have rallied other lesser groups to their banner. As such, ever have there been factions. Some, kinder than others."`
 			`	They look at you meaningfully. "There was a factional shift in progress here before you arrived. A little over a decade ago, for the first time in a century, a new anarcho-syndicalist faction from Freedom took over the control of the spaceport and reclaimed the mantle of our original founders who called themselves 'Base Principles'. While we as a city didn't particularly care, there were some who sympathized with them more than most. As overlords went they were, fair enough. After only a decade, and despite significantly greater access to resources than their predecessors, their control was already slipping. Deviants and extremists who lacked the capacity to subscribe to a nuanced view had supplanted most of their major players. If they did not succeed in their quest imminently, they were already doomed to be replaced. Still, they were nevertheless exploitative overlords, and we don't pity them... but their goals were more noble than most.`
-			`	"I tell you all this because I suspect you must seek out this one leader who is unaccounted for, and you should do so with all the knowledge at your disposal. The one leader who is not here was the last major player from the new 'Base Principles'. I do not know where your loyalties lie, so I ask you do you wish to know?"`
+			`	"I tell you all this because I suspect you must seek out this one leader who is unaccounted for, and you should do so with all the knowledge at your disposal. The one leader who is not here was the last major player from the new 'Base Principles'. I do not know where your loyalties lie, so I ask you: why do you wish to know?"`
+				goto reason
+			label impatient
+			`	They sigh. "The one leader who is not here was the last major player from the new 'Base Principles'. I do not know where your loyalties lie, so I ask you do you wish to know?"`
```

Comment:
```suggestion
			`	They sigh. "The one leader who is not here was the last major player from the new 'Base Principles'. I do not know where your loyalties lie, so I ask you: do you wish to know?"`
```

Reply:
This should be "why do you wish to know"

Reply:
It now reads, "I ask you: why do you wish to know"